### DISCUSSION TITLE
CHANGELOG.md versions have wrong markdown

### DISCUSSION THREAD

**Discussion by HarrySky** [Kludex/uvicorn#1388] [Potential Issue] [answered]
Hey, just noticed, that Renovate can't parse your `CHANGELOG.md` since `0.17.4` version :sweat_smile:

You seem to have wrong title style applied

**Actual**:
```
### 0.17.5 - 2022-02-16
...
### 0.17.4 - 2022-02-04
```

**Expected**:
```
## 0.17.5 - 2022-02-16
...
## 0.17.4 - 2022-02-04
```

**Comment by euri10** [comment, marked as answer]
good catch yes @HarrySky , raised as issue
https://github.com/encode/uvicorn/issues/1389
